### PR TITLE
L2  tests pass

### PR DESCRIPTION
**Pull request by VaclavK**

### CODE REVIEW

**Comment by VaclavK** [comment]
@djfr @arturek @dguerin

File: src/Tests/EShopWorld.Tools.Tests/AzScanDNSCLITests.cs [hunk 1]
```diff
@@ -82,5 +85,39 @@ internal static void CheckSecrets(IList<SecretBundle> secrets)
                 s.Value.Equals("http://5.5.5.5",
                     StringComparison.OrdinalIgnoreCase));
         }
+
+        // ReSharper disable once InconsistentNaming
+        private static void CheckSecretsEUS(IList<SecretBundle> secrets)
```
Comment (djfr) [inline src/Tests/EShopWorld.Tools.Tests/AzScanDNSCLITests.cs]: I wouldn't move test checks into a private method, but instead extend fluent assertions with a `HaveSecret` method after the `Should` and use that in the tests themselves.

File: README.md [hunk 2]
```diff
@@ -162,3 +166,9 @@ Scans Service bus namespaces and projects following secrets
 ```
 SB--{resourceName}--PrimaryConnectionString
 ```
+
+#### Testing strategy
+
+This repo uses layered test approach. There are level 0, 1 and 2 tests. Layer 0 runs tests for several components with mocked dependencies. Layer 1 runs tests call external dependencies but do not require deployment. Level 2 runs tests calling external dependencies and requires the tool to be deployed (installed).
```
Comment (dguerin) [inline README.md]: If there are layer 1 tests, they  need to be marked as such. Don't see the layer 1 in the test explorer in VS

File: src/EShopWorld.Tools/EShopWorld.Tools.csproj [hunk 3]
```diff
@@ -11,7 +11,7 @@
     <ToolCommandName>dotnet-esw</ToolCommandName>
     <RuntimeIdentifiers>win-x64</RuntimeIdentifiers>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
-    <Version>1.1.10</Version>
+    <Version>1.1.11</Version>
```
Comment (dguerin) [inline src/EShopWorld.Tools/EShopWorld.Tools.csproj]: Given the later updates to this, i won't be against updating this to 1.2.0. Given our constant discussion on this, this just a suggestion.

File: src/EShopWorld.Tools/Helpers/StringExtensions.cs [hunk 4]
```diff
@@ -32,29 +32,28 @@ public static string ToCamelCase(this string input)
         /// <param name="input">input values</param>
         /// <param name="suffixes">suffixes to strip out</param>
         /// <returns></returns>
-        public static string StripRecognizedSuffix(this string input, params string[] suffixes)
+        public static string EswTrim(this string input, params string[] suffixes)
```
Comment (dguerin) [inline src/EShopWorld.Tools/Helpers/StringExtensions.cs]: Very nice

File: src/EShopWorld.Tools/Commands/AzScan/AzScanCommandBase.cs [hunk 5]
```diff
@@ -50,13 +50,45 @@ protected AzScanCommandBase(Azure.IAuthenticated authenticated, KeyVaultClient k
...
+                        $"esw-{Domain.ToLowerInvariant()}-{EnvironmentName.ToLowerInvariant()}-{r.ToRegionCode().ToLowerInvariant()}"
+                    },
+                    Region = r
+                });
+
+        internal ResourceGroupDescriptor DomainResourceGroup => new ResourceGroupDescriptor
+        {
+            Name = $"{Domain.ToLowerInvariant()}-{EnvironmentName.ToLowerInvariant()}",
+            TargetKeyVaults = RegionList.Select(r =>
+                $"esw-{Domain.ToLowerInvariant()}-{EnvironmentName.ToLowerInvariant()}-{r.ToRegionCode().ToLowerInvariant()}")
```
Comment (arturek) [inline src/EShopWorld.Tools/Commands/AzScan/AzScanCommandBase.cs]: maybe one ToLowerInvariant at the end would be better? The same two lines above.

**Pull request merged**